Cargo.toml

```diff
@@ -57,10 +57,9 @@ url = "1.1"
 core-foundation = { version = "0.4.4", features = ["mac_os_10_7_support"] }
 
 [target.'cfg(windows)'.dependencies]
-kernel32-sys = "0.2"
 miow = "0.2"
-psapi-sys = "0.1"
-winapi = "0.2"
+winapi = { version = "0.3", features = ["handleapi", "jobapi", "jobapi2", "minwindef", "ntdef", 
+    "processenv", "processthreadsapi", "psapi", "synchapi", "winerror", "winbase", "wincon", "winnt"] }
```

Comment:
Out of curiosity, are all of these really required? I would have no idea how to generate this list...

Reply:
They should be required, yes.
Usually I just refactor the function calls to the new import path and then add the features when the import cannot be resolved.

Reply:
In order to reduce compile times I had to gate each header with a feature. Unfortunately this does mean you sometimes end up with really long feature lists like this, but there's really not any better way to deal with such a massive crate which will only continue to grow.

Reply:
Is there no better recourse for this? This seems pretty unergonomic to list dozens of modules in each project importing `winapi` and for example from an API like [`AssignProcessToJobObject`](https://msdn.microsoft.com/en-us/library/windows/desktop/ms681949(v=vs.85).aspx) how would I conclude it's in the `jobapi2` feature and module without reading both sets of documentation? Or is reading both sets of documentation required?

Reply:
Unfortunately items have to be defined in separate modules because sometimes two headers will define a symbol with the same name but different values, which also means I can't even glob import all the items into the crate root. The recommended way to figure out which module/feature an item comes from is using the search bar in the [documentation](https://docs.rs/winapi/*/x86_64-pc-windows-msvc/winapi/). I certainly welcome proposals for how to make things easier to use.